indexer/src/configs.rs

```diff
@@ -34,6 +36,15 @@ pub(crate) struct Opts {
     /// Port to enable metrics/health service
     #[clap(long, short, env, default_value_t = 3030)]
     pub port: u16,
+    /// S3 bucket name
+    #[clap(long)]
+    pub bucket: String,
```

Comment:
This makes these parameters required which is not a desired behavior for us. It's actually required only when the `custom` subcommand is fired.

I see two possible options here:

1. Make these 'Option<String>` and add corresponding checks when the `Custom` is provided and print relevant errors about missing parameters.
2. Make a copy of the `StartOptions` and require these parameters there.

I guess the second variant is preferable.